documentation_builder/simulating.ipynb

```diff
@@ -758,13 +759,14 @@
       "PRODUCING REACTIONS -- Pyruvate\n",
       "-------------------------------\n",
       "  %             FLUX   RXN ID                        REACTION                       \n",
-      " 50.0%   6.13 ± 6.13      PYK                  adp_c + h_c + pep_c --> atp_c + pyr_c\n",
-      " 50.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
+      " 85.0%   9.76 ± 0.24   GLCpts                     glc__D_e + pep_c --> g6p_c + pyr_c\n",
```

Comment:
So right now `summary` methods don't trigger an FBA solution, pFBA or otherwise. You can certainly get pFBA solutions in the summary methods, as long as the model is optimized prior to calling the summaries. This is partly by design, as they are supposed to be quick ways of examining a model's current state.  

This is muddled somewhat by the addition of the FVA options, which do trigger optimizations.